editor/src/components/inspector/sections/component-section/cartouche-control.tsx

```diff
@@ -28,6 +46,7 @@ export const IdentifierExpressionCartoucheControl = React.memo(
         onDelete={onDeleteCartouche}
         testId={testId}
         inverted={false}
+        contentIsComingFromServer={isDataComingFromHookResult}
```

Comment:
Surely this should be `contentIsComingFromAHook`?

Reply:
I added it based on the instructions in the original issue https://github.com/concrete-utopia/utopia/issues/5556